tests/src/Simple/PInvoke/PInvokeNative.Windows.targets

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+***********************************************************************************************
+Microsoft.NETCore.Native.Windows.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+This file defines Windows-specific steps in the build process for native AOT compilation.
+
+Licensed to the .NET Foundation under one or more agreements.
+The .NET Foundation licenses this file to you under the MIT license.
+See the LICENSE file in the project root for more information.
+***********************************************************************************************
+-->
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="Build" DependsOnTargets="Compile;Link"> </Target>
+  <Target Name="Compile"
+      Inputs="@(ClCompile)"
+      Outputs="$(PInvokeNativeObject)">
+
+    <ItemGroup>
+      <CompilerArg Include="/I$(IlcPath)\inc" />
+      <CompilerArg Include="@(ClCompile)" />
+      <CompilerArg Include="/Fo$(PInvokeNativeObject)" />
+      <CompilerArg Condition="'$(Configuration)' == 'Debug'" Include="/Od" />
+      <CompilerArg Condition="'$(Configuration)' != 'Debug'" Include="/O2" />
+      <CompilerArg Include="/c /nologo /W3 /GS /DCPPCODEGEN /EHs /Zi" />
+    </ItemGroup>
+ 
+    <MakeDir Directories="$(IntermediateOutputPath)" />
+    <WriteLinesToFile File="$(IntermediateOutputPath)pinvoke_cl.rsp" Lines="@(CompilerArg)" Overwrite="true" />
+
+    <Exec Command="cl @&quot;$(IntermediateOutputPath)pinvoke_cl.rsp&quot;">
+    </Exec>
+  </Target>
+  
+
+  <Target Name="Link"
+      Inputs="$(PInvokeNativeObject)"
+      Outputs="$(PInvokeNativeBinary)"
+      DependsOnTargets="Compile">
+
+    <ItemGroup>
+      <NativeLibrary Condition="$(NativeCodeGen) == ''" Include="$(IlcPath)\sdk\bootstrapper.lib" />
+      <NativeLibrary Condition="$(NativeCodeGen) == ''" Include="$(IlcPath)\sdk\Runtime.lib" />
```

Comment:
This is replicating a lot of the default targets  @files. Is there a way to restructure it to avoid this duplication?